### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by JISock** [chensong1995/HybridPose#3]
Hi Chen,

First of all, thank you for sharing your great work. I am a bit confused with the Dataset set-up
 in your description. You have stated that you have corrected Linemod pose labels, does that mean you used the labels different from the original Linemod dataset to evaluate the performance? I have downloaded the file in the link provided but could not find the corrected pose label(orientation and translation).

Thank you

**Comment by chensong1995** [comment]
Hello JISock,

Thanks for your interest in our research!

The "correction" to Linemod pose labels happens here: https://github.com/chensong1995/HybridPose/blob/master/lib/datasets/linemod.py#L187

We "shift" the translation component of the pose by a constant offset, and the offset is defined as the offset from the 3D model provided in the original Linemod dataset to the 3D model provided in "Blender meshes".

At the very initial phase of this project, I chatted with Sida Peng, who is the lead author of PVNet, one of our baseline approaches. I had a bug in my code at that time, and he offered me a few suggestions, including correcting the pose labels in the way described above. This correction is also part of the implementation of PVNet, as can be seen here: https://github.com/zju3dv/pvnet/blob/master/lib/utils/base_utils.py#L110

In the end, a combination of all his suggestions fixed the bug, and I have not looked into the pose correction issue since then.

I checked the code again today. It turns out this correction to translation vectors hardly has any real effect. If you set a breakpoint at https://github.com/chensong1995/HybridPose/blob/master/lib/datasets/linemod.py#L187, you will find that the offset `self.alignment_translation[object_name]` is very close to zero. Adding this offset to the original translation vector should not affect the pose and/or the final result in any meaningful way.

In short, we made a mistake. The original pose labels in Linemod are good enough, and we should not have corrected these labels. Luckily, our "correction" only very slightly changes the pose labels, and is well within the tolerance of human labeling error. I have taken down the section you mentioned in README.md.

I hope this helps,
Chen

**Comment by JISock** [comment]
Thanks Chen for your prompt reply, it was helpful.

**Comment by chensong1995** [comment]
Hello JISock,

This issue has been fixed now. Please take a look at my newest commit. Thanks again for pointing out the problem!

Best,
Chen
